plugins/aws/fix_plugin_aws/resource/backup.py

```diff
@@ -771,7 +972,7 @@ def add_tags(restore_job: AwsBackupRestoreJob) -> None:
                 "list-tags",
                 "Tags",
                 expected_errors=["ResourceNotFoundException"],
-                ResourceArn=restore_job.arn,
+                ResourceArn=restore_job.recovery_point_arn,
```

Comment:
Same here.
You would query for tags of the recovery point - that's wrong isn't it?
Does a job even have tags?